package.json

```diff
@@ -41,14 +41,20 @@
     "turbo": "2.1.2"
   },
   "devDependencies": {
+    "@eslint/js": "~9.10.0",
+    "@types/eslint__js": "~8.42.3",
     "commitizen": "4.3.0",
     "cz-conventional-changelog": "3.3.0",
+    "eslint": "~9.10.0",
+    "eslint-import-resolver-typescript": "~3.6.3",
+    "eslint-plugin-import-x": "~4.2.1",
+    "eslint-plugin-no-relative-import-paths": "~1.5.5",
     "prettier": "3.3.3",
-    "prettier-plugin-tailwindcss": "0.6.6"
+    "prettier-plugin-tailwindcss": "0.6.6",
+    "typescript": "~5.5.4",
```

Comment:
No TypeScript on base config 🙏

Reply:
Isn't Typescript used in all our projects? Can you explain me why this is wrong?

Reply:
Yes-ish, but in different ways. Each package has its own way of declaring TypeScript and its own rules.

The i18n package, for example has barely any TS besides of the index.d.ts and we don't compile TypeScript.

Reply:
got it, tks